python/ray/worker.py

```diff
@@ -73,24 +73,6 @@ def id(self):
 contained_objectids = []
```

Comment:
I suppose we could also get rid of `contained_objectids` at this point (we haven't been using it for a while).

It may also occur somewhere else in this file.